data/korath/korath missions.txt

```diff
@@ -257,3 +257,598 @@ mission "Small Scale Delivery"
+			`	This is the perfect location for life, so it's no surprise there are small plants and a few fish in the water. The seas are calmer here than the rest of the planet and an unusual formation in the ionosphere protects this spot from the worst rays of the nova. The dominant land-based lifeform seems to be toothed worms about the size of your arm.`
+			`	The strangest feature is what appears to be slow-moving islands. As one comes closer, its form is more clear: an unholy marriage between a blue whale and a giant squid. On the far end of the mesa, you spy a gravity crane system that looks suitable for carrying such a beast out of the water.`
+				decline
+
+
+
+mission "Korath [F04]: Fishy Delay"
+	invisible
+	landing
+	to offer
+		has "First Contact: Korath Exiles: offered"
+
+		# Require a successful landing on Rabasot Pa'ah, since that gives you the whale picture.
+		has "landed on Rabasot Pa'ah"
+
+		# You have to meet Iris and know she has knowledge of Korath:
+		has "Korath [F02]: A Fishy Proposition: done"
+	on offer
+		event "Korath [F04]: Fishy Delay Event" 5 14
+		fail
+
+event "Korath [F04]: Fishy Delay Event"
+
+
+
+mission "Korath [F05]: Fishy Reasoning"
+	landing
+	name "Seek answers from Iris"
+	description "Visit Iris at <planet> and ask her about Korath, fish transfers, an ocean planet, and her rainbow feather."
+	clearance "You explain the purpose of your visit, and the authorities inform you that Iris will expect you."
+	source
+		not planet "La'ab Bamen"
+	destination "La'ab Bamen"
+	to offer
+		has "event: Korath [F04]: Fishy Delay Event"
+	on offer
+		conversation
+			`While skimming damage and transit logs from your recent trips, a few old log entries catch your eye.`
+			`	When the Korath were drawing systems, three seemed to jump out at you. You now know they're Misam, home of New Tortuga; Atik, where the La'ab Bamen black market resides; and Kor Ak'Mari, the ocean planet's system. The first fish you transferred from New Tortuga to La'ab Bamen looked like a tiny relative of the island-sized fish on the ocean planet.`
+			`	Iris had quite a lot of knowledge of the Korath, and spoke of them with admiration. Most curious of all is the rainbow feather on her necklace. It was the exact shape of feathers on the Korath of Far'en Lei, but far more colorful.`
+			`	You last saw Iris on <planet>. Perhaps it is time to seek answers.`
+				accept
+
+
+mission "Korath [F06]: The Rainbow Feather"
+	landing
+	invisible
+	source "La'ab Bamen"
+	to offer
+		has "Korath [F05]: Fishy Reasoning: offered"
+	on offer
+		fail
+		conversation
+			`You find Iris in one of the narrow access tunnels of La'ab Bamen, observing the fish behind the glass.`
+			`	"I was wondering when you'd seek me out," she says, with a worried look. "You gave those two patrols quite a scare on Far'en Lei. They had never seen a human before, and they found one armed at their jail's lock."`
```

Comment:
Captain Last brought a hand weapon.

Reply:
I have revised the text so you only get that message if you pull out your gun. Otherwise, there are two other messages she'll use, depending on whether you offered food.